content/source/docs/cloud/sentinel/import/index.html.md

```diff
@@ -9,7 +9,7 @@ Sentinel Policies for Terraform are defined using the [Sentinel policy
 language](https://docs.hashicorp.com/sentinel/language/). A policy can include
 [imports](https://docs.hashicorp.com/sentinel/concepts/imports) which enable a
 policy to access reusable libraries, external data and functions. Terraform
-Enterprise provides three imports to define policy rules for the configuration,
+Enterprise provides four imports to define policy rules for the configuration,
```

Comment:
Given the addition of the tfrun import and the ordering of the four imports, would it be better to write the following?

"Terraform Enterprise provides four imports to define policy rules for the plan, configuration, and state associated with runs."

Also, should references to "Terraform Enterprise" be changed to "Terraform Cloud"?

Reply:
When I was originally searching for TFE references to update, I missed several that were broken up by hard wrap, like this one. (Great example of why hard wrap is awful for English prose.) Then I fixed it upstream... which is why there are now merge conflicts again in this branch. I'll go fix that.